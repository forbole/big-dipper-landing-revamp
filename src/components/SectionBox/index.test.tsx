import { render, screen } from '@testing-library/react';
import React from 'react';
import { MockTheme } from '~tests/mocks';
import Component from '.';

// ==================================
// unit tests
// ==================================
describe('SectionBox', () => {
  it('matches snapshot', async () => {
    const { container } = render(
      <MockTheme>
        <Component>
          <div>HELLO WORLD</div>
        </Component>
      </MockTheme>
    );
    expect(screen.getByText(/HELLO WORLD/i)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
});
