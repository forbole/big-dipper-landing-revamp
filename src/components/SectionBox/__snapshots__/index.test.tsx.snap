// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionBox matches snapshot 1`] = `
.emotion-0 {
  overflow: visible;
  padding: 0;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  min-width: 296px;
}

.emotion-0.sectionbox__main {
  padding: 24px;
}

@media (min-width:768px) {
  .emotion-0.sectionbox__main {
    padding: 64px 0px;
  }
}

<div>
  <section
    class="sectionbox__container emotion-0"
  >
    <div>
      HELLO WORLD
    </div>
  </section>
</div>
`;
